tests/test_settings.py

```diff
@@ -87,3 +87,44 @@ def test_update_settings(self, figures_env_tokens, run_celery):
             assert self.CELERY_TASK_NAME not in settings.CELERYBEAT_SCHEDULE
 
         assert settings.ENV_TOKENS['FIGURES'] == figures_env_tokens
+
+
+class TestDailyMauPipelineSettings(object):
+    """Tests MAU pipeline settings
+
+    See how edx-platforms does plugins. App: openedx.core.djangoapps.plugins
+
+    * constants.PLUGIN_APP_SETTINGS_FUNC_NAME
+    * openedx.core.djangoapps.plugins.plugin_settings
+
+    TODO: Abstract the core functionality in this test class because it presents
+    a pattern for testing plugin CeleryBeat schedule settings
+    """
+    TASK_NAME = 'figures-daily-mau'
+    TASK_FUNC = 'figures.tasks.populate_all_mau'
+    @pytest.fixture(autouse=True)
```

Comment:
An empty line is good to have.

```suggestion

    @pytest.fixture(autouse=True)
```